src/loc.rs

```diff
@@ -92,29 +89,28 @@ where
 }
 
 #[inline(always)]
-fn init(node: &Node, stats: &mut Stats, is_func_space: bool, is_unit: bool) -> usize {
+fn init(node: &Node, stats: &mut Stats, is_func_space: bool, is_unit: bool) -> (usize, usize) {
     let start = node.start_position().row;
+    let end = node.end_position().row;
 
     if is_func_space {
-        let end = node.end_position().row;
-
         stats.start = start;
         stats.end = end;
         stats.unit = is_unit;
     }
-    start
+    (start, end)
 }
 
 impl Loc for PythonCode {
     fn compute(node: &Node, _code: &[u8], stats: &mut Stats, is_func_space: bool, is_unit: bool) {
         use Python::*;
 
-        let start = init(node, stats, is_func_space, is_unit);
+        let (start, end) = init(node, stats, is_func_space, is_unit);
 
         match node.kind_id().into() {
             String | DQUOTE | DQUOTE2 | ExpressionStatement | Block | Module => {}
             Comment => {
-                stats.comment_lines.insert(start);
+                stats.comment_lines += (end - start) + 1;
```

Comment:
you probably need to handle the case of multiline comments:
https://twitter.com/gvanrossum/status/112670605505077248

Reply:
I added a `TODO` in the `Python` test because I don't know how to discriminate a string used as comment from a simple string

Reply:
An idea could be to look for string where the parent is an ExpressionStatement. I'd say that in the case the string is the statement and so this the only thing we've so a comment. Wdyt ?

Reply:
Yep, good idea! I'm going to implement that, thanks @calixteman! :)